konch.py

```diff
@@ -400,8 +400,10 @@ def out_prompt_tokens(self, *args, **kwargs):
                 if isinstance(output, (str, bytes)):
                     return [(Token.OutPrompt, output)]
                 else:
-                    return prompt
+                    return output
 
+        if isinstance(prompt, IPython.terminal.prompts.Prompts):
```

Comment:
The `class CustomPrompt` declaration should be moved to the `else` here.

Reply:
@sloria Yes sorry I hadn't noticed that that also really bothers `mypy`. I'll fix in a bit and push back up your way.